tutorials/Explanation.ipynb

```diff
@@ -43,7 +43,8 @@
     "- [MAE](./Mean_Absolute_Error.ipynb) / [RMSE](./Root_Mean_Squared_Error.ipynb) / [MSE](./Mean_Squared_Error.ipynb)\n",
     "- [Murphy Diagrams](./Murphy_Diagrams.ipynb)\n",
     "- [Pearson's Correlation](./Pearsons_Correlation.ipynb)\n",
-    "- [Quantile Loss](./Quantile_Loss.ipynb)\n"
+    "- [Quantile Loss](./Quantile_Loss.ipynb)\n",
+    "- [Bias](./Bias.ipynb)"
```

Comment:
Maybe rename the notebook to "Additive_Bias_and_Multiplicative_Bias". My concern about it being called "Bias" is that people might think it's a tutorial about bias in general.

Reply:
Done